google_compute_engine/metadata_scripts/tests/script_retriever_test.py

```diff
@@ -34,6 +34,18 @@ def setUp(self):
         self.mock_logger, self.script_type)
 
   @mock.patch('google_compute_engine.metadata_scripts.script_retriever.subprocess.check_call')
+  def testDownloadGsNoExec(self, mock_call):
+    mock_call.side_effect = subprocess.CalledProcessError('foo', 'bar')
+    gs_url = 'gs://fake/url'
+    self.assertIsNone(self.retriever._DownloadGsUrl(gs_url, self.dest_dir))
+    mock_call.assert_called_once_with(
```

Comment:
Thanks for the update. Current style is indenting by 2 everywhere. Indenting by 4 is reserved for breaking up a line for a line continuation. We have an open issue to move the code to pep 8.